source/computer/wsl-setup.rst

```diff
@@ -1,18 +1,138 @@
 WSL 配置指南
 ============
 
-.. note::
+:本节贡献者: |赵志远|\（作者）、
+             |田冬冬|\（审稿）、
+             |姚家园|\（审稿）
+:最近更新日期: 2021-01-17
+:预计花费时间: 120 分钟
+
+WSL 是什么?
+-------------
+
+Windows Subsystem for Linux（简称 WSL）是一个在 Windows 10 上能够运行原生
+Linux 二进制可执行文件（ ELF 格式）的兼容层，也被称为“适用于 Linux 的 Windows 子系统”。
+
+简单来说，在 Windows 10 上启用 WSL 功能后，就可以在 Windows 上安装 Linux 发行版，
+原样运行 Linux 下的大多数命令行工具、实用工具和应用程序。对于不想安装双系统或者虚拟机的
+用户来说是个不错的选择。
+
+WSL 可以同时安装多个不同的 Linux 发行版，并且随时进行切换，支持利用 ``wsl --export`` 
+命令将整个子系统随时打包备份，并在之后利用 ``wsl --import`` 命令进行还原。与虚拟机相比，
+在启动速度、消耗资源以及性能利用上有较大优势。
+
+WSL 分为 WSL1 和 WSL2 两个发行版本。由于底层原理不同，在大多数情况下，建议使用 WSL2，
+因为它提供更快的性能和 100% 的系统调用兼容性。仅当你的项目需要经常从 Linux 上访问存储
+在 Windows 系统上的文件或者利用 Windows 软件对 Linux 上的文件进行处理时，通过 WSL1 可实现更快的性能。
+
+官方目前没有弃用 WSL1 的计划，并且支持将任何一个已经安装的 Linux 发行版更改为 WSL1 或者 WSL2。
+
+安装 WSL
+---------------
+
+WSL 的安装方式可以参考官方文档：
+
+- 中文指南：https://docs.microsoft.com/zh-cn/windows/wsl/install-win10
+- 英文指南：https://docs.microsoft.com/en-us/windows/wsl/install-win10
+
+官方文档对 WSL 安装配置中可能遇到的常见问题都给出了详细的解决方案，请认真阅读。
+
+WSL 可以安装不同的 Linux 发行版，但目前官方并未提供 Fedora 发行版的安装。
+推荐使用  WSL 安装 Ubuntu 20.04 LTS 或 Ubuntu 20.10。
+配置 Linux
+-----------
+
+按照官方文档安装 WSL 之后，就可以根据你的 Linux 发行版进行配置使用了。
 
-   尚未完成。
+如果你安装了 Ubuntu，可以参考 《\ :doc:`/computer/ubuntu-setup` 》进行配置。
 
-   WSL 的安装方式可以参考：
+图形界面显示
+--------------
 
-   - 中文指南：https://blog.walterlv.com/post/how-to-install-wsl2.html
-   - 英文指南：https://docs.microsoft.com/en-us/windows/wsl/install-win10
+由于 WSL 本身不支持图形界面，我们需要额外安装 X Server 并指定图形输出位置，使得带有图形界面的
+桌面程序可以被显示和运行。
+
+Windows 上多种可选的 X Server 软件，`VcXsrv <https://sourceforge.net/projects/vcxsrv/>`__ 
+作为开源免费软件，使用方式简单，本文以此软件为例进行介绍。
```

Comment:
```suggestion
Windows 下常见的 X11 有 `VcXsrv <https://sourceforge.net/projects/vcxsrv/>`__\ 、
`Xming <http://www.straightrunning.com/XmingNotes/>`__\ 、
`Xmanager <https://www.xshellcn.com/>`__ 等。
其中，VcXsrv 是开源免费软件；Xming 和 Xmanager 是收费软件。
Xming 在 2007 年发布了最后一个免费版本（6.9.0.31）。
VcXsrv 的使用方式和界面与 Xming 极为相近。
推荐使用 VcXsrv，本文以此软件为例进行介绍。
```

Reply:
Are they really X11? X11 are installed on Linux, but these software are all installed on Windows.

Reply:
It seems that they are call X11 server.

![image](https://user-images.githubusercontent.com/50591376/104870936-bc060b80-5984-11eb-912d-1908540ddb5c.png)
![image](https://user-images.githubusercontent.com/50591376/104871033-ff607a00-5984-11eb-91d1-f6f58483f192.png)
![image](https://user-images.githubusercontent.com/50591376/104871048-0a1b0f00-5985-11eb-98e4-24f14c3615cf.png)

Reply:
I think it's better to name it X server.

Reply:
It does not matter. We can name it X Server while adding a note in the brackets.